test/unit/Cover/initialize.js

```diff
@@ -1,28 +1,60 @@
 const { expect } = require('chai');
+const { ethers } = require('hardhat');
 
 describe('initialize', function () {
-  it('should edit purchased cover and increase amount', async function () {
-    const { cover, accounts } = this;
-
-    // reset initialization
-    await cover.connect(accounts.governanceContracts[0]).updateUintParameters([0], ['0']);
+  it('should initialize variables correctly', async function () {
+    const { quotationData, stakingPool, futureCoverNFTAddress, coverAddress, coverUtilsLib } = this;
+    const Cover = await ethers.getContractFactory('Cover', {
+      libraries: {
+        CoverUtilsLib: coverUtilsLib.address,
+      },
+    });
+
+    // Uninitialized cover
+    const cover = await Cover.deploy(
+      quotationData.address,
```

Comment:
these values

quotationData.address
productsV1.address

are private fields. However we can try using getStorageAt https://docs.ethers.io/v5/api/providers/provider/#Provider-getStorageAt

to get the field value.

Let's try asserting them regardless using getStorageAt